geps/gep-1713.md

```diff
@@ -0,0 +1,220 @@
+
+
+See [GEP-1867](https://github.com/kubernetes-sigs/gateway-api/pull/1868) for more use cases of `infrastructure`.
+
+#### Go
+
+```go
+type GatewaySpec struct {
+  // Infrastructure defines infrastructure level attributes about this Gateway instance.
+  Infrastructure GatewayInfrastructure `json:"infrastructure"`
+  // ...
+}
+
+type GatewayInfrastructure struct {
+  // AttachTo allows the Gateway to associate itself with another resource.
+  // A common scenario is to reference another Gateway which marks
+  // this Gateway a child of another.
+  AttachTo LocalObjectReference `json:"attachTo"`
+}
+```
+
+#### YAML
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: child-gateway
+  labels:
+    gateway.networking.k8s.io/parent-gateway: parent-gateway
+spec:
+  gatewayClassName: example
+  infrastructure:
+    attachTo:
+      name: parent-gateway
+      kind: Gateway
+      group: gateway.networking.k8s.io
+  listeners:
+  - name: metrics
+    port: 8080
+    protocol: HTTP
+```
+
+### Semantics
+
+
+#### Gateway Attaching
+
+Gateways that _do not_ reference another Gateway (using `spec.infrastructure.attachTo`) are considered "primary" Gateways. A "child" Gateway is one where `spec.infrastructure.attachTo` field is set to a "primary" Gateway. A "sibling" is a Gateway that shares a "parent" with another "child" Gateway.
+
+
+Failure to attach a "child" Gateway to another MUST result in the "child" Gateway setting their `Accepted` condition to `False`. This MUST happen in the following scenarios:
+- Attaching to another "child" Gateway
+- Attaching to oneself (self-referential)
+- Attaching to a non-existing Gateway
+- Attaching to a Gateway with a different `GatewayClassName` 
+
+If a Gateway's `Accepted` condition is `False` then the conditions 'Programmed' and 'Ready' (if supported) MUST be set to `False`.
+
+#### Route Attaching
+
+Routes MUST be able to specify a "child" Gateways as a `parentRef` and make use of the fields in `ParentReference` to help target a specific listener.
+
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: httproute-example
+spec:
+  parentRefs:
+  - name: acme-lb
+    sectionName: https
+```
+
+#### `parent-gateway` label
+
+Implementations MUST label "child" Gateways with the `gateway.networking.k8s.io/parent-gateway` label. The value of the label is the name of the "parent" Gateway and it MUST be equivalent to the value `spec.infrastructure.attachTo.name`
+
+Users MUST NOT be able to edit this field.
+
+#### Merging Spec Fields
+
+
+#### GatewayClassName 
+
+The `GatewayClassName` MUST be set on "child" and "primary" Gateways and be equivalent.
+
+#### Listeners
+
+Implementations MUST treat the "primary" Gateway as having the concatenated list of all listeners from "child" Gateways.
+
+Validation of this list of listeners is subject to the discussion here: https://github.com/kubernetes-sigs/gateway-api/issues/1842
```

Comment:
not sure the issue covers this; we need to specify precedence for conflicts, like we do with routes

Reply:
Added a section for precedence and conflicts